homeassistant/components/fibaro/__init__.py

```diff
@@ -176,35 +176,18 @@ def _map_device_to_platform(device: DeviceModel) -> Platform | None:
             platform = Platform.LIGHT
         return platform
 
-    def _create_device_info(
-        self, device: DeviceModel, devices: list[DeviceModel]
-    ) -> None:
-        """Create the device info. Unrooted entities are directly shown below the home center."""
-
-        # The home center is always id 1 (z-wave primary controller)
-        if device.parent_fibaro_id <= 1:
-            return
+    def _create_device_info(self, master_device: DeviceModel) -> None:
```

Comment:
Can we give it a better name like main device or hub device?

Reply:
Sure, I renamed it to main_device.